api/schema.graphql

```diff
@@ -952,7 +952,7 @@ type ExecuteUserProgramPayload {
   outcome, which could be one of:
   - Compile error
   - Runtime error
-  - Failed execution (incorrect input/output at termination)
+  - Rejected execution (incorrect input/output at termination)
   - Success
```

Comment:
Really minor but missed accepted here

Reply:
Yeah ik I was just gonna leave that one cause I don't think it makes a diff

Reply:
good enough for me lol